server/src/mikro-orm.config.ts

```diff
@@ -32,20 +33,19 @@ class CustomNamingStrategy extends AbstractNamingStrategy implements NamingStrat
   }
 }
 
+const clientUrl = process.env.DATABASE_URL
+
 // TODO: import connection-related properties from root .env
 const mikroORMConfig: Options = {
   // debug: process.env.NODE_ENV !== 'production',
+  clientUrl,
   debug: true,
-  dbName: 'clubs',
-  host: 'db', // must match service name of postgres container in docker-compose
-  user: 'loolabs',
-  password: 'loolabs',
-  port: 5432,
   highlighter: new SqlHighlighter(),
   entities: ['**/*.entity.js'],
   entitiesTs: ['**/*.entity.ts'], // path to your TS entities (source), relative to `baseDir`
   metadataProvider: TsMorphMetadataProvider,
   migrations: {
+    disableForeignKeys: false,
```

Comment:
Heroku dyno had similar issues to https://github.com/mikro-orm/mikro-orm/issues/190 (postgres replica stuff??) when starting up. The solution proposed is to add this option in Mikro ORM configuration. I guess it enables foreign key checks on migrations which is good??

![image](https://user-images.githubusercontent.com/14998054/106179548-3135c580-6169-11eb-978c-a0a5e37bf936.png)